frontend/src/pages/stakingOverview/StakingDetails.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Typography, Grid } from '@mui/material';
+import { Chains } from './Chains';
+
+const StakingDetails = (props) => {
+  return (
+    <Chains chains={props.chains} />
```

Comment:
ditto here. This component has single element. Move it to parent component.

Reply:
done